common-test.dotty/src/main/scala/org/scalatest/SharedHelpers.scala

```diff
@@ -1769,7 +1769,9 @@ object SharedHelpers extends Assertions with LineNumberHelper {
   // This gives a comparator that compares based on the value in the passed in order map
   private def orderMapComparator[T](orderMap: Map[T, Int]): java.util.Comparator[T] =
     new java.util.Comparator[T] {
-      def compare(x: T, y: T): Int = {
+      def compare(_x: T | Null, _y: T | Null): Int = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.